### ISSUE TITLE
[Bug]: Need to upgrade etcd version

### ISSUE THREAD

**Issue by xiaofan-luan** [milvus-io/milvus#15443]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

etcd may crash due to the issue (https://github.com/etcd-io/etcd/issues/13119)

### Expected Behavior

need to upgrade etcd reliability to a version slightly higher than 3.5.0

**Comment by yanliang567** [comment]
@xiaofan-luan do we have a recommended version here?

**Comment by xiaofan-luan** [comment]
not sure, 3.5.2 is released two days ago and that might be a good choice.

**Comment by scipe** [comment]
Can you put that on hold for now? My search not works properly with latest version of etcd. I am not sure yet why. But this is one biggest changes in last time that I did.

**Comment by xiaofan-luan** [comment]
sure, @LoveEachDay please also help investigate on that.

**Comment by xiaofan-luan** [comment]
> @xiaofan-luan I think we unblocked here

Any etcd version recommended? @scipe

**Comment by scipe** [comment]
@xiaofan-luan we planned to upload 200M (with metadata) vectors on next week. So I can not say smth about my version of etcd. So far its works.

**Comment by LoveEachDay** [comment]
We'll trigger a stability test for 100m data for milvus cluster with etcd 3.5.1.
